src/test/java/util/LottoTestHelper.java

```diff
@@ -0,0 +1,16 @@
+package util;
+
+import domain.LottoNumber;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTestHelper {
+  public static List<LottoNumber> parse(String numbers) {
+    return Arrays.stream(numbers.split(","))
+        .map(String::trim)
+        .map(LottoNumber::new)
+        .collect(Collectors.toList());
+  }
```

Comment:
```suggestion
  public static List<LottoNumber> lottoNumbersFixture(int... numbers) {
    return Arrays.stream(numbers)
            .mapToObj(LottoNumber::new)
            .collect(Collectors.toList());
  }
```

```java
Lotto lotto = new Lotto(lottoNumbersFixture(42, 41, 40, 10, 20, 30));
```
int를 받는 생성자를 사용하면 되지 않을까요?  
매번 테스트하기 위해 문자열로 감쌀 필요가 없어질 것 같아요

Reply:
구조까지 설명해주셔서 이해 되었습니다. 👍

애당초 `List<LottoNumber>` 리스트 타입으로 받고 있어
제가 생각한 `List<Integer>` 생성자 오버로딩 자체가 불가능하겠네요 🥲

--- 

테스트 픽스처 부분은 문자열을 가공하는 것 보다 LottoNumber int 생성자를 적극적으로 사용하시면 좋을 것 같습니다!